src/index.js

```diff
@@ -1,36 +1,79 @@
 import './style.css';
 
-const toDoItems = [
-  {
-    info: 'Wash the dishes',
-    completed: true,
-    index: 1,
-  },
-  {
-    info: 'Set Appointment for Tomorrow',
-    completed: true,
-    index: 2,
-  },
-  {
-    info: 'Work on  my Thesis',
+const form = document.querySelector('form');
+const input = document.querySelector('.input');
+const todoContainer = document.querySelector('.todoContainer');
+
+let todo;
+let todos = JSON.parse(localStorage.getItem('todos')) || [];
+const store = () => {
+  todo = {
+    Description: input.value,
+    id: todos.length,
     completed: false,
```

Comment:
- [ ] You have added the functionality of add todo properly. :clap: but it would be nice if your todo id should start from 1 instead of 0 which is a crucial requirement in your project :thinking: